### PR TITLE
Add trim, trim_left, trim_right

### PR DESCRIPTION
**Pull request by anderseknert**
Signed-off-by: Anders Eknert <anders@eknert.com>

### CODE REVIEW

File: src/main/clojure/jarl/builtins/registry.clj [hunk 1]
```diff
@@ -18,7 +18,10 @@
    "split" strings/builtin-split
    "startswith" strings/builtin-startswith
    "substring" strings/builtin-substring
+   "trim" strings/builtin-trim
+   "trim_left" strings/builtin-trim-left
    "trim_prefix" strings/builtin-trim-prefix
+   "trim_right" strings/builtin-trim-right
```
Comment (johanfylling) [inline src/main/clojure/jarl/builtins/registry.clj:24]: When we have all string built-ins, it'd be good to also add a parser test, with a policy that simply exercise all these built-ins in one go, end-to-end.

**Review by johanfylling — APPROVED** [review]
LGTM 👍

File: src/main/clojure/jarl/builtins/strings.clj [hunk 2]
```diff
@@ -4,6 +4,16 @@
   (:import (se.fylling.jarl BuiltinException)
            (java.util.regex Pattern)))
 
+(defn- trim-left [s cutset]
```
[the remark this answers is not in the document]
Comment (anderseknert) [reply src/main/clojure/jarl/builtins/strings.clj:7]: Yep! I don't think they're truly "private", but at least it shows the intent.

File: src/main/clojure/jarl/builtins/registry.clj (hunk 1 above)
Comment (anderseknert) [reply src/main/clojure/jarl/builtins/registry.clj:24]: Definitely! Could use some help on that 🤓

**Pull request merged**